distribution/ddf-common/src/main/resources/common-bin.xml

```diff
@@ -51,6 +51,26 @@
                 <exclude>RELEASE-NOTES</exclude>
                 <exclude>karaf-manual*.html</exclude>
                 <exclude>karaf-manual*.pdf</exclude>
+
+                <!-- excluding jars we don't need that have OWASP vulnerabilities -->
+                <!--
+                Excluding this unused jar, as we don't use it (TO BE CONFIRMED)
+                and it's associated with CVE-2004-0009
+                -->
+                <exclude>**/bundles/org.apache.servicemix.bundles.not-yet-commons-ssl/**</exclude>
```

Comment:
We do use this jar and it is needed for our certificate UI to work correctly. It is embedded within that jar, however, so it shouldn't be getting deployed anyways.